evalml/automl/automl_search.py

```diff
@@ -538,8 +556,8 @@ def _find_best_pipeline(self):
                     train_indices = self.data_splitter.transform_sample(X_train, y_train)
                     X_train = X_train.iloc[train_indices]
                     y_train = y_train.iloc[train_indices]
-                best_pipeline = self._engine.train_pipeline(best_pipeline, X_train, y_train,
-                                                            self.optimize_thresholds, self.objective)
+                best_pipeline = train_pipeline(best_pipeline, X_train, y_train,
```

Comment:
i wonder if we want this to still be called through the engine?

a use case to imagine is that you are running evalml on your laptop and you've connected it to a remote dask cluster that has plenty of compute (perhaps the `X` and `y` are reference to dask dataframes where the data is remote too). you want all the training to happen on the remote dask cluster. 

if i understand the code correctly, right now the search phase will happen on the remote cluster, but the final training of the pipeline would happen locally.

Reply:
Good call! That was my intention but I think I forgot to actually make the change hehe.